utils/src/test/kotlin/ExtensionsTest.kt

```diff
@@ -86,6 +86,17 @@ class ExtensionsTest : WordSpec({
         }
     }
 
+    "String.stripUserNameFromUrl" should {
+        "strip the user name from a string representing a URL" {
+            "ssh://bot@gerrit.host.com:29418/parent/project".stripUserNameFromUrl() shouldBe
+                    "ssh://gerrit.host.com:29418/parent/project"
+        }
+
+        "not modify a string not representing a URL" {
+            "This is not a URL".stripUserNameFromUrl() shouldBe "This is not a URL"
+        }
+    }
```

Comment:
guess its out of scope of ort, does it need to be considered?: what happens if the url contains the password in addition to username separated by ':'

Reply:
It's not out of scope, I've added a test to verify the password is also stripped.